WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -515,7 +547,9 @@ private extension OrderListViewController {
 extension OrderListViewController: UITableViewDelegate {
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        tableView.deselectRow(at: indexPath, animated: true)
+        if splitViewController?.isCollapsed == true {
+            tableView.deselectRow(at: indexPath, animated: true)
+        }
```

Comment:
I noticed that the cell remain selected after rotating back in portrait:
<img src="https://user-images.githubusercontent.com/495617/157279549-73103f39-3aec-4558-af52-8c6ba391c6d7.png" width=300 />

Reply:
This has been fixed in 3132df8a4de80b33efac478d545ba73d14c9224f by updating the highlight state on `viewDidLayoutSubview`.